autoload/compe/documentation.vim

```diff
@@ -7,6 +7,7 @@ let s:FloatingWindow = vital#compe#import('VS.Vim.Window.FloatingWindow')
 let s:window = s:FloatingWindow.new()
 call s:window.set_var('&wrap', 1)
 call s:window.set_var('&conceallevel', 2)
+call s:window.set_var('nvim_compe_documentation', 1)
```

Comment:
I think `nvim_` prefix should be removed.

Reply:
Thank you for the quick review!